lib/debug/session.rb

```diff
@@ -1159,10 +1159,15 @@ def cancel_auto_continue
     end
 
     def show_help arg
+      require_relative "color"
+      extend Color
```

Comment:
This method can be called multiple times and putting this will result it to can `extend Color` repeatedly as well, which should be avoided.

Reply:
I created an include Color above class Session this time, I think it is in a better place.

Thank you.